tests/test_particleContainer.py

```diff
@@ -40,6 +40,25 @@ def particle_container(Npart, std_geometry, distmap, boxarr, std_real_box):
 
     iseed = 1
     pc.InitRandom(Npart, iseed, myt, False, std_real_box)
+
+    # add runtime components: 1 real 2 int
+    pc.add_real_comp(True)
+    pc.add_int_comp(True)
+    pc.add_int_comp(True)
+
+    #   can be removed after
+    #     https://github.com/AMReX-Codes/amrex/pull/3615
+    pc.resize_runtime_real_comp(1, True)
+    pc.resize_runtime_int_comp(2, True)
```

Comment:
This pattern can be simplified with the introduction of
https://github.com/AMReX-Codes/amrex/pull/3615

Reply:
done.